config/config.go

```diff
@@ -109,6 +109,7 @@ const (
 type Experimental struct {
 	EnableStargz         bool `toml:"enable_stargz"`
 	EnableReferrerDetect bool `toml:"enable_referrer_detect"`
+	EnableConfigSource   bool `toml:"enable_config_source"`
```

Comment:
Can we have a doc or comments telling the configuration item is used for?

Reply:
In addition, it will be great if we can add an example configuration item at `misc/snapshotter/config.toml` with a usage comment

Reply:
I will add an example

Reply:
> Can we have a doc or comments telling the configuration item is used for?

This option is for compatibility with older versions of nydusd, which require a config drop disk